main.html

```diff
@@ -0,0 +1,178 @@
+        <div class="Rectangle">
+          <div class="col-md-3">
+            <div class="rect">
+              <img src="sun.png" alt="" width="142" height="142">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <h1 class="htext">Super Flight to Mars
+                        </h1>
+            <h7 class="stext">Two earth Years</h7>
+            <h1 class="ftext">Race through the Solar System to reach the Red planet.  This flight will take 11 Earth Years for each side of the journey.</h1>
+          </div>
+          <div class="col-md-3">
+            <h1 class="d_money">$18,200</h1>
+            <button type="button" class="bbtn1">BOOK TRIP</button>
+          </div>
+        </div>
+        
+                <div class="Rectangle">
+          <div class="col-md-3">
+            <div class="rect">
+              <img src="bee.png" alt="" width="142" height="142">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <h1 class="htext">Super Flight to Mars
+                        </h1>
+            <h7 class="stext">Two earth Years</h7>
+            <h1 class="ftext">Race through the Solar System to reach the Red planet.  This flight will take 11 Earth Years for each side of the journey.</h1>
+          </div>
+          <div class="col-md-3">
+            <h1 class="d_money">$18,200</h1>
+            <button type="button" class="bbtn1">BOOK TRIP</button>
+          </div>
+        </div>
+
+        <div class="Rectangle">
+          <div class="col-md-3">
+            <div class="rect">
+              <img src="sun.png" alt="" width="142" height="142">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <h1 class="htext">Super Flight to Mars
+                        </h1>
+            <h7 class="stext">Two earth Years</h7>
+            <h1 class="ftext">Race through the Solar System to reach the Red planet.  This flight will take 11 Earth Years for each side of the journey.</h1>
+          </div>
+          <div class="col-md-3">
+            <h1 class="d_money">$18,200</h1>
+            <button type="button" class="bbtn1">BOOK TRIP</button>
+          </div>
+        </div>
+        
+                <div class="Rectangle">
+          <div class="col-md-3">
+            <div class="rect">
+              <img src="bee.png" alt="" width="142" height="142">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <h1 class="htext">Super Flight to Mars
+                        </h1>
+            <h7 class="stext">Two earth Years</h7>
+            <h1 class="ftext">Race through the Solar System to reach the Red planet.  This flight will take 11 Earth Years for each side of the journey.</h1>
+          </div>
+          <div class="col-md-3">
+            <h1 class="d_money">$18,200</h1>
+            <button type="button" class="bbtn1">BOOK TRIP</button>
+          </div>
+        </div>
+
+        <div class="Rectangle">
```

Comment:
read up about semantic naming of CSS classes and improve your naming.

Reply:
Done.